tests/Skoruba.IdentityServer4.Admin.IntegrationTests/Common/HttpClientExtensions.cs

```diff
@@ -1,16 +1,19 @@
 ﻿using System.Net.Http;
+using Skoruba.IdentityServer4.Admin.Common.Settings;
 using Skoruba.IdentityServer4.Admin.Constants;
 using Skoruba.IdentityServer4.Admin.Middlewares;
 
 namespace Skoruba.IdentityServer4.Admin.IntegrationTests.Common
 {
     public static class HttpClientExtensions
     {
+        static AdminAppSettings _defaultSettings = new AdminAppSettings();
```

Comment:
Any idea - how to read the appsettings from json? Because you can use another values in JSON file and another as the default then tests will be failed.